lib/main_common.pm

```diff
@@ -548,4 +549,11 @@ sub load_filesystem_tests() {
     }
 }
 
+sub load_iso_in_external_tests() {
```

Comment:
see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/3281 for reference. Better remove all parentheses